CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog TYPO3 Crawler widget
+
+## Version 0.0.1
```

Comment:
Personally I prefer "Unreleased" and adjust it accordingly to the version when publishing the extension. That makes it clearer that the features are unreleased yet.

Reply:
I changed this to `## Unreleased version 0.0.1`